packages/manager/apps/telecom/src/app/telecom/telephony/service/contact/contact.controller.js

```diff
@@ -382,10 +382,12 @@ export default class TelecomTelephonyServiceContactCtrl {
       this.directoryForm.PJSocialNomination = this.directoryForm.socialNomination;
     }
 
-    this.directoryForm.wayNumberExtra = this.directoryForm.wayNumberExtra.replace(
-      /&nbsp;/g,
-      '',
-    );
+    if (this.directoryForm.wayNumberExtra) {
+      this.directoryForm.wayNumberExtra = this.directoryForm.wayNumberExtra.replace(
+        /&nbsp;/g,
+        '',
```

Comment:
don't you want it to be a space ' ' ?

Reply:
As I didn't change this part (I just added the test), I choose to keep the same behaviour